integration-tests/react-percy-storybook/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "react-percy-storybook-integration-tests",
+  "version": "1.0.0",
+  "scripts": {
+    "storybook": "start-storybook -p 9001 -c .storybook -s ./.storybook/public",
+    "storybook:build": "build-storybook -s ./.storybook/public",
+    "storybook:percy-debug": "percy-storybook --debug",
+    "storybook:percy": "build-storybook -s ./.storybook/public && percy-storybook --widths=320,1280 --rtl_regex=Direction",
+    "storybook:percy-no-build": "percy-storybook --widths=320,1280 --rtl_regex=Direction"
+  },
+  "dependencies": {
+    "faker": "^4.1.0",
+    "mockdate": "^2.0.1",
+    "prop-types": "^15.5.10",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-match-media": "^2.0.1"
```

Comment:
Now that we're using Yarn workspaces to install all the dependencies, the integration test project dependencies are getting installed in all the CI jobs, not just whatever job is running the storybook snapshots. This particular package says it requires Node 5+, so installing blows up in the Node 4 job. Marking it as an optional dependency allows installing to succeed, and since this project isn't currently run in Node 4 it doesn't actually matter that it's not installed there.